zyte_spider_templates/utils.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+from scrapy.utils.url import parse_url
+
+
+def get_domain(url: str) -> str:
+    allowed_domain = parse_url(url).netloc
```

Comment:
Can you please rename the var as the code doesn't look specific to allowed domains?

Reply:
Good call. https://github.com/zytedata/zyte-spider-templates/pull/27/commits/c66f4ddb9a59577f6046e8ab27375c5e0ac9f4f8